hack/istio/install-istio-via-istioctl.sh

```diff
@@ -425,16 +425,21 @@ if [ "${REDUCE_RESOURCES}" == "true" ]; then
     --set components.pilot.k8s.resources.requests.memory=1Mi"
 fi
 
+if [ "${CLUSTER_NAME}" != "" ]; then
+     CLUSTER_NAME_OPTION=" \
+       --set values.global.multiCluster.clusterName=${CLUSTER_NAME}"
+   fi
```

Comment:
Just some reformatting.. The "fi" is indented incorrectly and no need for a new line in the cluster name option value.
```suggestion
if [ "${CLUSTER_NAME}" != "" ]; then
  CLUSTER_NAME_OPTION="--set values.global.multiCluster.clusterName=${CLUSTER_NAME}"
fi
```